packages/design-system/src/components/Layout/SkipLinks/SkipLinks.tsx

```diff
@@ -18,16 +18,18 @@ const SkipLinks = ({ nav, main }: SkipLinksProps) => {
 			<ul role="list">
 				{nav && (
 					<li key="nav">
-						<Link className="skip-links__link" href="#nav">
-							{t('SKIP_LINK_NAVIGATION', 'Go to navigation')}
-						</Link>
+						{/* eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex */}
```

Comment:
I had to add a span to hide the link, because I can't hide the link without overriding its styles, which I shouldn't do. 
But adding a TabIndex on a span (to make the link visible through keyboard nav) is not ok by the linter as spans aren't interactive elements.